extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/util/CronHelper.java

```diff
@@ -0,0 +1,128 @@
+/**
```

Comment:
Should this class not be moved to the scheduler implementation for everyone to use?